sgm/inference/helpers.py

```diff
@@ -84,6 +83,14 @@ def load(self, model: torch.nn.Module):
         """
         return model.to(self.device)
 
+    def autocast(self):
```

Comment:
Borrowed from #49 to fix autocast, which it turns out would have broken if you passed a real device name in, too.